recipes/supervisord.rb

```diff
@@ -0,0 +1,42 @@
+#
+# Cookbook Name:: datadog
+# Recipe:: supervisord
+#
+# Copyright 2011-2015, Datadog
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+include_recipe 'datadog::dd-agent'
+
+# Build a data structure with configuration.
```

Comment:
Can you add a link to the actual template in dd-agent? Something like this: https://github.com/DataDog/chef-datadog/blob/master/recipes/jmx.rb#L4